softwareComponents/simplesimServerLib/include/simplesim_server.hpp

```diff
@@ -34,6 +34,11 @@ class SimplesimServerOpts {
                 .desc( "Python packet filter file" );
 
         cli.opt( &verbose, "v verbose" ).desc( "Run simulator in verbose mode" );
+
+        cli.opt( &collision, "c collision", 1 )
+                .desc( "Collision model to use:\n"
+                    "0 - no collision detection\n"
+                    "1 - simple component unit sphere collision");
```

Comment:
Instead of meaningless number you should use choice: https://gknowles.github.io/dimcli/v7.1.1/guide.html#choice

Reply:
Also, I agree with @Ondraceq that the collision check should be the default.

Reply:
Done, I have added an enum for collision models into rofiworld.hpp. Maybe a bit of an overkill as of now, but perhaps there will also be the AABB collision or soemthing else someday.